Example/PatchKit/Info.plist

```diff
@@ -28,6 +28,8 @@
 	<array>
 		<string>armv7</string>
 	</array>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>Wanna steal ur data</string>
```

Comment:
we should prob change this lOL

Reply:
lolll ill take it out. Added a github issue to add this to the actual pod (this was just for example project)